src/App.tsx

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
 import './App.scss';
 
-class App extends React.Component {
-  addOne = () => {};
+type State = {
+  count: number,
+};
 
-  add100 = () => {};
+class App extends React.Component<{}, State> {
+  state:State = {
+    count: 0,
+  };
 
-  increase = () => {};
+  addToCounter = (num = 1):void => {
```

Comment:
не рекомендую прописывать явно, если совсем ничего не возвращаешь
```suggestion
  addToCounter = (num = 1) => {
```